.github/workflows/stale-issues.yml

```
@@ -17,7 +17,6 @@ name: Close inactive issues
 on:
   schedule:
     - cron: "30 1 * * *"
```

```suggestion
    - cron: "30 1 * * *"

```

Keep that empty line, it does not make sense to remove